--- POE-0.38/Makefile.PL~	2006-10-27 22:58:42.000000000 +0200
+++ POE-0.38/Makefile.PL	2006-10-27 22:58:42.000000000 +0200
@@ -49,12 +49,12 @@
 
 $prompt .= "(Any text other than Y or y will be taken as a no)";
 
-my $ret = "n";
+my $ret = "y";
 if (grep /^--default$/, @ARGV) {
-  print $prompt, " [n] n\n\n";
+  print $prompt, " [y] y\n\n";
 }
 else {
-  $ret = prompt($prompt, "n");
+  $ret = prompt($prompt, "y");
 }
 
 my $marker = 'run_network_tests';
--- POE-0.38/tests/90_regression/cfedde-filter-httpd.t~	2006-10-28 09:02:21.000000000 +0200
+++ POE-0.38/tests/90_regression/cfedde-filter-httpd.t	2006-10-28 09:02:21.000000000 +0200
@@ -11,6 +11,10 @@
     print "1..0 # skip - HTTP::Request needed to test POE::Filter::HTTPD\n";
     exit;
   }
+  unless (-f 'run_network_tests') {
+    print "1..0 # skip - Network access (and permission) required to run this test\n";
+    exit;
+  }
 }
 
 use Test::More tests => 2;
--- POE-0.38/tests/90_regression/immute-server-tcp-filter.t~	2006-10-28 09:02:43.000000000 +0200
+++ POE-0.38/tests/90_regression/immute-server-tcp-filter.t	2006-10-28 09:02:43.000000000 +0200
@@ -11,6 +11,10 @@
     print "1..0 # skip - HTTP::Request needed to test POE::Filter::HTTPD\n";
     exit;
   }
+  unless (-f 'run_network_tests') {
+    print "1..0 # skip - Network access (and permission) required to run this test\n";
+    exit;
+  }
 }
 
 #BEGIN { @INC = ('/share/immute/svn/poe/poe/lib', @INC); }
