Index: lib/POE/Filter/HTTPD.pm
===================================================================
--- lib/POE/Filter/HTTPD.pm	(revision 2222)
+++ lib/POE/Filter/HTTPD.pm	(working copy)
@@ -175,7 +175,11 @@
 
   # Use the request line to create a request object.
 
-  my $r = HTTP::Request->new($1, URI->new($2));
+  my $method = uc $1;
+  my $req_path = $2;
+  $req_path =~ s/^[\/]{2,}/\//; # fix double slash starting path
+
+  my $r = HTTP::Request->new($method, URI->new($req_path));
   $r->protocol($proto);
   $self->[CLIENT_PROTO] = $proto = _http_version($proto);
 
@@ -206,7 +210,6 @@
   # If this is a GET or HEAD request, we won't be expecting a message
   # body.  Finish up.
 
-  my $method = uc $r->method();
   if ($method eq 'GET' or $method eq 'HEAD') {
     $self->[FINISH]++;
     # We are sending this back, so won't need it anymore.
